client/signup/jetpack-connect/controller.js

```diff
@@ -207,4 +246,29 @@ export default {
 			context.store
 		);
 	},
+
+	plansPreSelection( context ) {
+		const Plans = require( './plans' ),
+			site = sites.getSelectedSite(),
```

Comment:
As an attempt to migrate away from `sites-list`, there is a group effort to remove usage of `sitesList.getSelectedSite()` - #8726. Let's get rid of it and use the corresponding redux ui selector for that purpose.